config/filament-authentication-log.php

```diff
@@ -22,4 +22,5 @@
         'direction' => 'desc',
     ],
 
+    'panel_id' => 'admin'
```

Comment:
This config can be removed as we will use the `Filament::getCurrentPanel()->getId()`